src/CMakeLists.txt

```diff
@@ -2,18 +2,23 @@
 set(Kokkos_DIR "$ENV{Kokkos_ROOT}" CACHE STRING "Kokkos root directory")
 find_package(Kokkos REQUIRED)
 find_package(LAPACK REQUIRED)
+find_package(KokkosKernels REQUIRED)
 
 target_sources(${oturb_exe_name} PRIVATE main.cpp)
+set_target_properties(${oturb_exe_name} PROPERTIES LINKER_LANGUAGE CXX)
 target_link_libraries(${oturb_exe_name} PRIVATE
     Kokkos::kokkos
+    Kokkos::kokkoskernels
```

Comment:
I didn't realize KokkosKernels is a separate package from kokkos. Is it a separate download/installation? Or is it just an option within kokkos that is not enabled by default?

Reply:
Yes, it's a separate package that links to your existing Kokkos.  https://github.com/kokkos/kokkos-kernels.  It also links to BLAS and magma for the linear solver on CPU and GPU.